examples/demo/src/orders/OrderList.tsx

```diff
@@ -152,169 +154,72 @@ const TabbedDatagrid = () => {
                             />
-                        </DatagridConfigurable>
+                        </OrdersTable>
                     )}
                     {filterValues.status === 'cancelled' && (
-                        <DatagridConfigurable
-                            rowClick="edit"
-                            omit={['total_ex_taxes', 'delivery_fees', 'taxes']}
-                            preferenceKey={preferenceKeys.cancelled}
-                        >
-                            <DateField source="date" showTime />
-                            <TextField source="reference" />
-                            <CustomerReferenceField source="customer_id" />
-                            <ReferenceField
-                                source="customer_id"
-                                reference="customers"
-                                link={false}
-                                label="resources.orders.fields.address"
-                            >
-                                <AddressField />
-                            </ReferenceField>
-                            <NumberField
-                                source="basket.length"
-                                label="resources.orders.fields.nb_items"
-                            />
-                            <NumberField
-                                source="total_ex_taxes"
-                                options={{
-                                    style: 'currency',
-                                    currency: 'USD',
-                                }}
-                            />
-                            <NumberField
-                                source="delivery_fees"
-                                options={{
-                                    style: 'currency',
-                                    currency: 'USD',
-                                }}
-                            />
-                            <NumberField
-                                source="taxes"
-                                options={{
-                                    style: 'currency',
-                                    currency: 'USD',
-                                }}
-                            />
-                            <NumberField
-                                source="total"
-                                options={{
-                                    style: 'currency',
-                                    currency: 'USD',
-                                }}
-                                sx={{ fontWeight: 'bold' }}
-                            />
-                        </DatagridConfigurable>
+                        <OrdersTable storeKey={storeKeyByStatus.cancelled} />
                     )}
                 </>
             )}
         </Fragment>
     );
 };
 
+const Column = DataTable.Col<Order>;
+const ColumnNumber = DataTable.NumberCol<Order>;
+
+const OrdersTable = React.memo(
```

Comment:
Nice component but it does not showcase Fragments to group columns that you can still use as the `DataTable` children which was impossible with `Datagrid` if I'm not mistaken. Your call though